### ISSUE TITLE
 DVTCoreDeviceEnabledState: DVTCoreDeviceEnabledState_Disabled set via user default (DVTEnableCoreDevice=disabled)

### ISSUE THREAD

**Issue by 390138244dys** [facebook/react-native#38263]
### Description

This problem occurred when my project was running yarn ios. I looked up a lot of information did not find what is the reason, hope you help to answer, thank you very much!

ding@admindeMacBook-Pro ios % yarn ios
yarn run v1.22.19
$ react-native run-ios
info Found Xcode workspace "BluetoothCarApp.xcworkspace"
info Found booted iPhone 13
info Launching iPhone 13
info Building (using "xcodebuild -workspace BluetoothCarApp.xcworkspace -configuration Debug -scheme BluetoothCarApp -destination id=253E8077-F1B8-46EF-95CF-EA5A32DB8671")
⠇ Building the app
⠋ Building the app
⠙ Building the app
⠙ Building the app

info 💡 Tip: Make sure that you have set up your development environment correctly, by running react-native doctor. To read more about doctor command visit: https://github.com/react-native-community/cli/blob/main/packages/cli-doctor/README.md#doctor 

error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening BluetoothCarApp.xcworkspace.
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace BluetoothCarApp.xcworkspace -configuration Debug -scheme BluetoothCarApp -destination id=253E8077-F1B8-46EF-95CF-EA5A32DB8671

User defaults from command line:
    IDEPackageSupportUseBuiltinSCM = YES

Prepare packages

Computing target dependency graph and provisioning inputs

Create build description
Build description signature: 5100ec3bec98b0dae481c3ff5612b055
Build description path: /Users/ding/Library/Developer/Xcode/DerivedData/BluetoothCarApp-emxhkcfnvinahzhjvrnpkxrhbmwt/Build/Intermediates.noindex/XCBuildData/5100ec3bec98b0dae481c3ff5612b055.xcbuilddata

note: Building targets in dependency order
error: Multiple commands produce '/Users/ding/Library/Developer/Xcode/DerivedData/BluetoothCarApp-emxhkcfnvinahzhjvrnpkxrhbmwt/Build/Products/Debug-iphonesimulator/BluetoothCarApp.app/Frameworks/glog.framework'
    note: That command depends on command in Target 'BluetoothCarApp' (project 'BluetoothCarApp'): script phase “[CP] Embed Pods Frameworks”
    note: That command depends on command in Target 'BluetoothCarApp' (project 'BluetoothCarApp'): script phase “[CP] Embed Pods Frameworks”
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'React-utils' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'React-NativeModulesApple' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'React-debug' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Flipper-PeerTalk' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'React-runtimescheduler' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'ReactCommon' from project 'Pods')
/Users/ding/Desktop/Wrok/SMW/Code/adult-master-2023/ios/Pods/Pods.xcodeproj: warning: Multiple targets match implicit dependency for linker flags '-framework folly'. Consider adding an explicit dependency on the intended target to resolve this ambiguity. (in target 'React-Core-AccessibilityResources' from project 'Pods')
    note: Target 'Flipper-Folly' (in project 'Pods')
    note: Target 'RCT-Folly' (in project 'Pods')
/Users/ding/Desktop/Wrok/SMW/Code/adult-master-2023/ios/BluetoothCarApp.xcodeproj: warning: Multiple targets match implicit dependency for linker flags '-framework fmt'. Consider adding an explicit dependency on the intended target to resolve this ambiguity. (in target 'BluetoothCarApp' from project 'BluetoothCarApp')
    note: Target 'Flipper-Fmt' (in project 'Pods')
    note: Target 'fmt' (in project 'Pods')
/Users/ding/Desktop/Wrok/SMW/Code/adult-master-2023/ios/BluetoothCarApp.xcodeproj: warning: Multiple targets match implicit dependency for linker flags '-framework folly'. Consider adding an explicit dependency on the intended target to resolve this ambiguity. (in target 'BluetoothCarApp' from project 'BluetoothCarApp')
    note: Target 'Flipper-Folly' (in project 'Pods')
    note: Target 'RCT-Folly' (in project 'Pods')
warning: Run script build phase 'Start Packager' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'BluetoothCarApp' from project 'BluetoothCarApp')
warning: Run script build phase 'Bundle React Native code and images' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'BluetoothCarApp' from project 'BluetoothCarApp')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'fmt' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Flipper-Folly' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'React-Codegen' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'RCT-Folly' from project 'Pods')

2023-07-09 20:33:44.564 xcodebuild[34430:338401] DVTCoreDeviceEnabledState: DVTCoreDeviceEnabledState_Disabled set via user default (DVTEnableCoreDevice=disabled)
** BUILD FAILED **

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

### React Native Version

6.8.2

### Output of `npx react-native info`

ding@admindeMacBook-Pro ios % npx react-native info
info Fetching system and libraries information...
System:
    OS: macOS 13.4.1
    CPU: (4) x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
    Memory: 41.91 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.16.1 - ~/.nvm/versions/node/v18.16.1/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v18.16.1/bin/yarn
    npm: 9.5.1 - ~/.nvm/versions/node/v18.16.1/bin/npm
    Watchman: 2023.07.03.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.12.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.4, iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4
    Android SDK: Not Found
  IDEs:
    Android Studio: 2022.2 AI-222.4459.24.2221.10121639
    Xcode: 14.3.1/14E300c - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.19 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: Not Found
    react-native: Not Found
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

### Steps to reproduce

DVTCoreDeviceEnabledState_Disabled set via user default (DVTEnableCoreDevice=disabled)

### Snack, code example, screenshot, or link to a repository

`require_relative '../node_modules/react-native/scripts/react_native_pods'
require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'

platform :ios, '12.4'
# //use_frameworks!是否使用依照原生项目
use_frameworks!
install! 'cocoapods', :deterministic_uuids => false

target 'BluetoothCarApp' do
  # //这里是react-native使用的pods（react-native 0.64+）
  config = use_native_modules!

  # Flags change depending on the env values.
  flags = get_default_flags()

  use_react_native!(
    # :path => config[:reactNativePath],
    # Hermes is now enabled by default. Disable by setting this flag to false.
    # Upcoming versions of React Native may rely on get_default_flags(), but
    # we make it explicit here to aid in the React Native upgrade process.
    :hermes_enabled => false,
    :fabric_enabled => flags[:fabric_enabled],
    # Enables Flipper.
    #
    # Note that if you have use_frameworks! enabled, Flipper will not work and
    # you should disable the next line.
    :flipper_configuration => FlipperConfiguration.enabled,
    # An absolute path to your application root.
    # :path => config[:reactNativePath],
    :app_path => "#{Pod::Config.instance.installation_root}/.."
  )

  target 'BluetoothCarAppTests' do
    inherit! :complete
    # pod 'hermes-runtime-darwin-v0.70.7', :path => './Locallib/hermes-runtime-darwin-v0.70.7'
    # Pods for testing
  end

  post_install do |installer|
    react_native_post_install(
      installer,
      # Set `mac_catalyst_enabled` to `true` in order to apply patches
      # necessary for Mac Catalyst builds
      :mac_catalyst_enabled => false
    )
    __apply_Xcode_12_5_M1_post_install_workaround(installer)
  end
  
  # //如果使用use_frameworks!需要添加这一部分，如果不使用则不需要（非常重要）
  pre_install do |installer|
    Pod::Installer::Xcode::TargetValidator.send(:define_method,
     :verify_no_static_framework_transitive_dependencies) {}
  end
end`

**Comment by cortinico** [comment]
Hi,
In this issue tracker we accept only issues that are fully in English.

I'm closing this for now, please consider translating it to English and reopening it.

Thanks.

**Comment by griffinbaker12** [comment]
Facing the same issue here

**Comment by codesalley** [comment]
> Facing the same issue here

Weird but works. specify the simulator example
```sh
 yarn ios --simulator="iPhone 12 Pro"
```

if you get an error like this `xcodebuild: error: Unable to find a destination matching the provided destination specifier`

- open project in xcode 
- `Window` -> `Devices and Simulators`
- set simulator `Show run destination` to `always`


still trying to understand why thou 😆

**Comment by talhatsk** [comment]
> > Facing the same issue here
> 
> Weird but works. specify the simulator example
> 
> ```shell
>  yarn ios --simulator="iPhone 12 Pro"
> ```
> 
> if you get an error like this `xcodebuild: error: Unable to find a destination matching the provided destination specifier`
> 
> * open project in xcode
> * `Window` -> `Devices and Simulators`
> * set simulator `Show run destination` to `always`
> 
> still trying to understand why thou 😆

it didn't work for me, anything else?

**Comment by talhatsk** [comment]
> Facing the same issue here

how did you fix the issue?

**Comment by alfangr** [comment]
In my case because of my Podfile.lock was not changed when I moved the branch for `branch-a` to `branch-b`, I solved it with `cd ios && pod install`.

**Comment by kafeg** [comment]
To keep this note here.

I found what helped me with xcodebuild. Just called the sample command before run xcodebuild.

Found solution here:

https://twitter.com/DonnyWals/status/1692508366109630888

If you've been struggling to get Xcode 14.3 to run apps on your iOS 17 devices try running the following command in terminal and then relaunch Xcode: defaults write com.apple.dt.Xcode DVTEnableCoreDevice enabled

Source: https://developer.apple.com/forums/thread/730947
